common/thanos/values.yaml

```diff
@@ -136,6 +153,51 @@ query:
     #   namespace: default
     #   port: 10901
 
+ruler:
+  enabled: false
+  # When a ThanosRuler deployment is paused, no actions except for deletion
+  # will be performed on the underlying objects.
+  paused: false
```

Comment:
Interesting, haven't looked into this yet and the explanation is straight from k8s. Not sure how it works though.